Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -2785,15 +2785,20 @@ void LocalDOMWindow::disableSuddenTermination()
         page->chrome().disableSuddenTermination();
 }
 
-LocalFrame* LocalDOMWindow::frame() const
+Frame* LocalDOMWindow::frame() const
+{
+    return localFrame();
```

Comment:
Is this just here for backwards compatibility? Maybe it would enforce better coding if we removed it?

Reply:
Yeah, I'll do a follow up.

Reply:
Actually, we can't get rid of this override because there are clients relying on `DOMWindow::frame` which is implemented by this class and `RemoteDOMWindow`.